Added parameter that allows to execute a script file

olivierg13 opened the pull request:
Hello @edenman 

I've seen that you merged a couple of pull requests recently.

This feature adds a --script parameter to spoon that allows to execute a script before the tests (in normal mode), or between each devices (in sequential mode).

I'm using myself this feature on a private deploy for my project for months now. I just did not have the time before to do a pull request. It allows me to reset my test server to zero before I run the test on each device connected to the platform (it's a ruby on rails server, so completely dissociated from the device).

I guess it can be used for other purposes too. Does this feature interest you?

I ran 'mvn install' and everything went fine (tests and checkstyle are passing).
Don't hesitate to answer if you want something changed in the code (naming, comments, logic etc...). I can easily do any change and squash it.

Thank you in advance.

Cheers,

Olivier

edenman commented on spoon-runner/src/main/java/com/squareup/spoon/SpoonRunner.java:
```diff
@@ -203,6 +212,43 @@ private SpoonSummary runTests(AndroidDebugBridge adb, Set<String> serials) {
+      try {
+        Runtime run = Runtime.getRuntime();
+        Process proc = run.exec(new String[] {
+                "/bin/bash", "-c", "echo 5 | " + script.getAbsolutePath()
```
why the `echo 5`?

olivierg13 replied:
That's an excellent question, I did not see that XD. I think it was for test purposes ^^
I'm gonna change that and rename the feature too (probably tomorrow).

Thank you.

Olivier

edenman commented:
Thanks for the PR!  Can you rename it to `--init-script` and `initScript` globally?

olivierg13 commented:
Hello @edenman 

I removed the 'echo 5' and changed the feature name by --init-script and initScript.
Tests are passing also.

Don't hesitate to tell me if you need something else?

Cheers

edenman commented:
Thanks @olivierg13!
